### PR TITLE
Cell Charger QoL - Remove with crowbar and drop cell onto to charge

### PR DESCRIPTION
**Pull request by JamieD1**
Cell Charger QoL - Remove with crowbar and drop cell onto to charge

:cl:  
tweak: Cell Charger QoL - Remove with crowbar and drop cell onto to charge
/:cl:

### CODE REVIEW

File: code/game/machinery/cell_charger.dm [hunk 1]
```diff
@@ -32,6 +32,12 @@
 		. += "<span class='notice'>The status display reads: Charge rate at <b>[charge_rate]J</b> per cycle.<span>"
 
 /obj/machinery/cell_charger/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/crowbar) && !panel_open)
+		if(!charging)
+			return
+		user.transferItemToLoc(charging, src))
```
Comment (Bibby0110) [inline code/game/machinery/cell_charger.dm]:
```suggestion
		user.put_in_hands(charging))
```

File: code/game/machinery/cell_charger.dm [hunk 2]
```diff
@@ -32,6 +32,12 @@
 		. += "<span class='notice'>The status display reads: Charge rate at <b>[charge_rate]J</b> per cycle.<span>"
 
 /obj/machinery/cell_charger/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/crowbar) && !panel_open)
+		if(!charging)
+			return
+		charging.forceMove(drop_location())
```
Comment (Bibby0110) [inline code/game/machinery/cell_charger.dm]: user.put_in_hands(charging)

Comment (Bibby0110) [inline code/game/machinery/cell_charger.dm]:
```suggestion
		user.put_in_hands(charging)
```

**Review by Bibby0110 — APPROVED** [review]

**Comment by ReddicusDragon** [comment]
Does this mean we need a crowbar to use a cell charger now?

**Pull request merged**